src/components/Example2.tsx

```
@@ -0,0 +1,371 @@
+    if (buttonName) {
+        if (buttonName.includes(draft)) {
+            tempStatus = 0;
+            setSavedChanges(true);
+        }
+    }
+    form.validateFields()
+        .then(() => {
+            data.stopPropagation();
+
+            let subtitles: SubtitleCreate[] = [];
+            if (subTitle != null && subTitle.subtitleText) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)